kodi_addon_checker/check_addon.py

```diff
@@ -124,3 +127,16 @@ def _get_addons(xml_url: str):
         LOGGER.error(errrt)
     except requests.exceptions.ConnectTimeout as errct:
         LOGGER.error(errct)
+
+
+def check_schema(report: Report, Addon_xml, schema_path):
+    schema = xmlschema.XMLSchema(schema_path)
+
+    try:
+        schema.validate(Addon_xml)
+        if schema.is_valid(Addon_xml):
```

Comment:
This if is is useless, either validate throws and exception or not. If it doesn't the xml is valid.

Reply:
So basically we don't want any output when the schema is valid ?

Reply:
I didn't say that, just saying if you hit this line you already know it is valid, because otherwise it you jump into the except block.